packages/admin/admin/src/form/fields/CurrencyField.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from "react";
+
+import { Field, FieldProps } from "../Field";
+import { FinalFormCurrencyInput } from "../FinalFormCurrencyInput";
+
+export const CurrencyField = ({ ...restProps }: FieldProps): React.ReactElement => {
```

Comment:
The return type can be inferred by TS

```suggestion
export const CurrencyField = ({ ...restProps }: FieldProps) => {
```